src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncIteratorTests.cs

```diff
@@ -347,7 +353,7 @@ await foreach (var s in M())
 ref struct S
 {
 }";
-            var comp = CreateCompilationWithTasksExtensions(new[] { source, s_common }, options: TestOptions.DebugExe);
+            var comp = CreateCompilationWithTasksExtensions(new[] { source, AsyncStreamsTypes }, options: TestOptions.DebugExe);
```

Comment:
All of these could use the helper I created `CreateCompilationWithAsyncIterator`. Up to you, though.